data/xml/P18.xml

```diff
@@ -2,7 +2,7 @@
 <collection id="P18">
   <volume id="1">
     <meta>
-      <booktitle>Proceedings of the 56th Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers)</booktitle>
+      <booktitle short="ACL (Long Papers)">Proceedings of the 56th Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers)</booktitle>
```

Comment:
ACL -> Proceedings / Proc. of ACL ?

Reply:
"Proceedings of" could always be added automatically, and I think there are many situations where one just wants "ACL".